frontend/src/features/comparisonSeries/ComparisonSeries.tsx

```diff
@@ -215,6 +220,24 @@ const ComparisonSeries = ({
     setDialogOpen(false);
   };
 
+  const skipTutorial = () => {
+    closeDialog();
+
+    setTutorialIsSkipped(true);
+    window.localStorage.setItem('tutorialIsSkipped', 'true');
+  };
+
+  const skipTutorialButton = (
+    <Button
+      sx={{ marginRight: '8px' }}
+      color="secondary"
+      variant="contained"
+      onClick={skipTutorial}
+    >
+      {t('tutorial.skip')}
```

Comment:
We can harmonize the translation key with the rest of the app: `t('tutorial.skipTheTutorial')`